.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug report
```

Comment:
Very minor: why are we rebranding issue template as bug report?

Reply:
that's part of the way new templates are done